src/js/angular/repositories/controllers.js

```diff
@@ -171,6 +174,18 @@ function LocationsAndRepositoriesCtrl($scope, $rootScope, $uibModal, toastr, $re
             .finally(() => $scope.loader = false);
     }
 
+    function getAndSetClusterStatus() {
+        ClusterRestService.getNodeStatus().then((response) => {
+            $scope.isInCluster = response.data.nodeState;
+        }).catch((error) => {
+            if (error.status === 404) {
+                $scope.isInCluster = false;
+            } else {
+                console.error('An unexpected error occurred:', error);
```

Comment:
I guess this is dumped in the log because otherwise it'd spam the user with toastr popups when not cluster. Right? But I'm not convinced that we should constantly check for cluster status here in the first place. We need to confirm that with @avataar